admin_guide/manage_users.adoc

```diff
@@ -9,84 +9,77 @@
 
 toc::[]
 
+[[managing-users-overview]]
+== Overview
+
+This topic describes the management of user accounts, including how new user
+accounts are created in {product-title} and how they can be deleted.
```

Comment:
Maybe somehow link to this arch topic in the Overview somewhere:

https://docs.openshift.com/enterprise/latest/architecture/core_concepts/projects_and_users.html